OpenSim/Simulation/Model/BushingForce.h

```diff
@@ -80,15 +75,28 @@ OpenSim_DECLARE_CONCRETE_OBJECT(BushingForce, Force);
 //==============================================================================
 // PUBLIC METHODS
 //==============================================================================
-    /** Default constructor leaves bodies unspecified, sets the bushing frames
+    /** Default constructor leaves frames unspecified, sets the bushing frames
     to be at their body origins, and sets all bushing parameters to zero. **/
     BushingForce();
-    /** This constructor provides everything needed to create a real 
-    bushing. See property declarations for more information. **/
-    BushingForce(const std::string& body1Name, 
+
+    /** Construct the BushingForce given the offset frames that it tries to
+    align according to the physical properties of the bushing.
+    See property declarations for more information. **/
+    BushingForce(const PhysicalOffsetFrame& frame1,
```

Comment:
> I targeted BushingForce to show how to use PhysicalOffsetFrames as subcomponents for the purpose of abstracting these frame connections. The BushingForce gets its dependencies (and connectors) by employing the PhysicalOffsetFrame. The offset is more than likely to be of interest to the BushingForce (only) and not necessarily the Body (or other PhysicalFrame) origin. It also does not preclude using the Body frame with an identity offset transform in that (no offset) case.

The alternative is to assume the offset exists somewhere (e.g. in the Model), but this does not map well to the use cases/examples. When you create a bushing force you generally also specify the offset (location, orientation in frame 1 and 2) then it makes more sense to be an attribute of the BusingForce (which is what I have done).

You are right, the only convenience constructor that is necessary is the one that takes the offset frames, since you can easily construct the PhysicalOffsetFrame first.

I will remove the others.